.gitignore

```diff
@@ -103,6 +103,7 @@ py38venv
 /site
 
 .idea/
+.vscode/
```

Comment:
Or we could include a standard .vscode so it's the same for everyone.

Reply:
We could, in some other repositories I included my project file for VS Code. But in the case of this project, at the moment I still think to leave out the specific file for VS Code. Formatting is enforced by CI validation anyway, and there is a Makefile, so it's not so important.